content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -0,0 +1,371 @@
+                    event.preventDefault();
+                    if (index > 0) {
+                        navigateAndFocusAccordionItem(index - 1);
+                    }
+                    break;
+                case keyCodes.ARROW_RIGHT:
+                case keyCodes.ARROW_DOWN:
+                    event.preventDefault();
+                    if (index < lastIndex) {
+                        navigateAndFocusAccordionItem(index + 1);
+                    }
+                    break;
+                case keyCodes.HOME:
+                    event.preventDefault();
+                    navigateAndFocusAccordionItem(0);
+                    break;
+                case keyCodes.END:
+                    event.preventDefault();
+                    navigateAndFocusAccordionItem(lastIndex);
+                    break;
+                default:
+                    return;
+            }
+        }
+
+        /**
+         * Refreshes the accordion item based on the current {@code Accordion#_toggle} index
+         * @private
+         */
+        function refreshAccordion() {
+            if (that._toggle >= 0 && that._elementItempanels && that._elementItems) {
+                if (!that._elementItempanels[that._toggle].classList.contains(selectors.expanded.itempanel)) {
+                    expandAccordionItem(that._elementItempanels[that._toggle], that._elementItems[that._toggle]);
+                } else {
+                    collapseAccordionItem(that._elementItempanels[that._toggle], that._elementItems[that._toggle]);
+                }
+            }
+        }
+
+        /**
+         * Collapses the provided accordion item.
+         * @private
+         * @param {Object} itempanel Provided accordion item's itempanel
+         * @param {Object} item Provided accordion item's item
+         */
+        function collapseAccordionItem(itempanel, item) {
+            if (itempanel && item) {
+                itempanel.classList.remove(selectors.expanded.itempanel);
+                itempanel.setAttribute("aria-hidden", true);
+                item.classList.remove(selectors.expanded.item);
+                item.setAttribute("aria-selected", false);
+                item.setAttribute("tabindex", "-1");
+                item.getElementsByClassName("cmp-accordion__item--title--icon")[0].classList.remove(selectors.expanded.icon);
+            }
+        }
+
+        /**
+         * Expands the provided accordion item.
+         * @private
+         * @param {Object} itempanel Provided accordion item's itempanel
+         * @param {Object} item Provided accordion item's item
+         */
+        function expandAccordionItem(itempanel, item) {
+            if (itempanel && item) {
+                itempanel.classList.add(selectors.expanded.itempanel);
+                itempanel.removeAttribute("aria-hidden");
+                item.classList.add(selectors.expanded.item);
+                item.setAttribute("aria-selected", true);
```

Comment:
Accordions should use aria-expanded instead of aria-selected (https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html)